STM32F405RGTx/Core/Src/rtos_threads/measurements_thread.cpp

```diff
@@ -5,12 +5,26 @@
 #include "mc.hpp"
 #include "cmsis_os.h"
 #include "threads.hpp"
+#include "lut.hpp"
 
-// TODO: add defines here for conversions..
+#define CURRENT_SENSE_RESISTANCE                1E-3
+#define ADC_TO_VOLTAGE(adc)                     ( adc * (3.3/(1 << 12)) )
+#define VOLTAGE_TO_CURRENT(voltage)             (voltage / CURRENT_SENSE_RESISTANCE)
+#define VOLTAGE_TO_RESISTANCE(voltage)          ( -10000 / (voltage - 3.3) )
+#define VOLTAGE_DIVIDER_CONVERSION(voltage)     ( voltage * 17.25 )
```

Comment:
Where does `17.25` come from? If it makes the code more readable, putting the calculation in the macro won't make it less efficient, since the calculation will be optimized out anyways.